plugins/network-elements/tungsten/pom.xml

```diff
@@ -130,5 +130,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-exec</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.6.1</version>
+        </dependency>
```

Comment:
do we really need this? we are trying to phase out all non log4j to than procy that and have a single point of upgrade. I would like to not introduce extra logging proxies/facades/frameworks

Reply:
No, not really. I removed that dependency.